cell.rb

```diff
@@ -1,22 +1,17 @@
 class Cell
   attr_accessor :status
   def initialize
-    @status = 'Dead'
+    @status = (rand(2)) == 1 ? true : false
```

Comment:
A shorter way could be `rand(2) == 1` Maybe you do not need to use the ternary operator, since the 'equal to == ' operator returns true or false.

Reply:
Fixed, thanks!